tools/functional-tester/etcd-checker/main.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2016 The etcd Authors
```

Comment:
The command will be used in the third step of the workflow: https://github.com/coreos/etcd/pull/6519#discussion_r80612145